lib/Backend/IR.h

```diff
@@ -50,6 +52,31 @@ struct CapturedValues
         Assert(refCount > 0);
         refCount++;
     }
+
+    void CopyTo(JitArenaAllocator *allocator, CapturedValues *other)
+    {
+        Assert(other != nullptr);
+        this->constantValues.CopyTo(allocator, other->constantValues);
+        this->copyPropSyms.CopyTo(allocator, other->copyPropSyms);
+
+        if (other->argObjSyms != nullptr)
+        {
+            other->argObjSyms->ClearAll();
+            JitAdelete(allocator, other->argObjSyms);
+        }
+
+        if (this->argObjSyms != nullptr)
+        {
+            other->argObjSyms = this->argObjSyms->CopyNew(allocator);
+        }
+        else
+        {
+            other->argObjSyms = nullptr;
+        }
+
+        // Ignore refCount because other objects might still reference it
+        other->refCount;
```

Comment:
What is the intention here?

Reply:
That's just to emphasize that I didn't forget the refCount :)

Reply:
the comment is good enough. Delete the code line

---
In reply to: [241617916](https://github.com/Microsoft/ChakraCore/pull/5861#discussion_r241617916) [](ancestors = 241617916)